src/App.js

```diff
@@ -67,6 +68,7 @@ class App extends Component {
                 path="/event/:address/admin"
                 component={SingleEventAdmin}
               />
+              <Route path="/user/:username" component={UserProfile} />
```

Comment:
Can we have link from somewhere? One idea is to link to /user/$username when clicking username from the header. If we do so, can you also add link to "Edit Profile" from the user profile page?

Reply:
Also can we replace the link from the list of participants which currently link to twitter address?